### PR TITLE
[11.0][MIG] mig auth dynamic groups

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

File: auth_dynamic_groups/views/res_groups.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<odoo>
+    <record model="ir.ui.view" id="view_groups_form">
```
Comment (tbaden) [inline auth_dynamic_groups/views/res_groups.xml]: optional:
id before model

**Review by tbaden — APPROVED** [review]
Thanks for the work 👍 
I did a code review: LGTM
If you like an optional change

**Comment by NL66278** [comment]
@tbaden Done!

**Comment by tbaden** [comment]
thanks 👍  don't forget to squash your commits at the end

**Comment by NL66278** [comment]
Most commits are just the history of the module. Basically I want to keep one migration commit, and one to contain the review fixes.

**Review by hbrunn — APPROVED** [review]

**Comment by NL66278** [comment]
Ping - I think this is ready to merge.... who has the authority?

**Pull request closed without merging**